### ISSUE TITLE
Error after "block removed" with Potgres

### ISSUE THREAD

**Issue by FilipPyrek** [AugurProject/augur-node#811]
After successfully getting augur-node synced, after a little while this error occures. And if I try to restart it, it appears again (but with different block number). 

```
new block: 7170949, 1549241454 (Mon Feb 04 2019 00:50:54 GMT+0000 (Coordinated Universal Time))
block removed: 7170949 (0x465dec49b71a65472c07e57c0747d50dd7757ea8fcdfe1a5fb4eac676e5a896f)
Fatal Error, shutting down servers { error: invalid input syntax for integer: "0x6d6b85"
    at Connection.parseE (/home/ubuntu/augur-node/node_modules/pg/lib/connection.js:546:11)
    at Connection.parseMessage (/home/ubuntu/augur-node/node_modules/pg/lib/connection.js:371:19)
    at Socket.<anonymous> (/home/ubuntu/augur-node/node_modules/pg/lib/connection.js:114:22)
    at Socket.emit (events.js:189:13)
    at Socket.EventEmitter.emit (domain.js:441:20)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
  name: 'error',
  length: 98,
  severity: 'ERROR',
  code: '22P02',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'numutils.c',
  line: '106',
  routine: 'pg_atoi' }
update "transactionHashes" set "removed" = $1 where "blockNumber" = $2 - invalid input syntax for integer: "0x6d6b85"
error: invalid input syntax for integer: "0x6d6b85"
    at Connection.parseE (/home/ubuntu/augur-node/node_modules/pg/lib/connection.js:546:11)
    at Connection.parseMessage (/home/ubuntu/augur-node/node_modules/pg/lib/connection.js:371:19)
    at Socket.<anonymous> (/home/ubuntu/augur-node/node_modules/pg/lib/connection.js:114:22)
    at Socket.emit (events.js:189:13)
    at Socket.EventEmitter.emit (domain.js:441:20)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
Stopping Augur Node Server
```

`blockNumber` is typed as integer. 
When I converted to decimal the number causing the error `0x6d6b85` I got `7170949` which is the number of the block where it failed. 
**So I guess there is a missing HEX to DEC conversion when "block removed" event happens.**

**Comment by pgebheim** [comment]
Heya Filip,

Yep, you're correct. Have a fix coming shortly if you can test a branch to make sure it's working alright for you.

**Comment by FilipPyrek** [comment]
@pgebheim great, thanks! 🙂

**Comment by epheph** [comment]
Fixed in #812 . Thanks @FilipPyrek !
